packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -528,18 +659,153 @@ abstract class SliderComponentShape {
   /// If [labelPainter] is non-null, then [labelPainter.paint] should be
   /// called with the location that the label should appear. If the labelPainter
   /// passed is null, then no label was supplied to the [Slider].
+  void paint(
+      PaintingContext context,
+      Offset center, {
+        Animation<double> activationAnimation,
+        Animation<double> enableAnimation,
+        bool isDiscrete,
+        TextPainter labelPainter,
+        RenderBox parentBox,
+        SliderThemeData sliderTheme,
+        TextDirection textDirection,
+        double value,
+      });
+}
+
+// The following shapes are the material defaults.
+
+/// This is the default shape to a [Slider]'s track if no other shape is
```

Comment:
Nit: First sentence in it's own paragraph.

https://www.dartlang.org/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph

Reply:
Still need to do this...

Reply:
Done